Jenkinsfile

```diff
@@ -2,34 +2,37 @@
 
 REPOSITORY = 'govuk-content-schemas'
 
+// second boolean parameter describes whether the app is using the central
+// buildProject script - if so it will report its success individually, so this
+// job does not need to wait for it.
 def dependentApplications = [
```

Comment:
Is it possible to use some sort of hash literal to avoid the need for a comment?

```groovy
[app: 'businesssupportfinder', usesBuildProject: false]
```

Perhaps that's too verbose though?